docs/src/upgrade/release-3.7.x.asciidoc

```diff
@@ -29,6 +29,18 @@ complete list of all the modifications that are part of this release.
 
 === Upgrading for Users
 
+==== Date manipulation functions
+
+Date manipulations in Gremlin queries were only possible using closures, which may or may not be supported by
+different providers. In 3.7.1, we introduce the `asDate()`, `dateAdd` and `dateDiff` steps aimed to replace the usage of closure.
+
+The following example demonstrates usage of newly introduced steps:
+
+[source,text]
+----
+gremlin> g.inject("2023-08-02T00:00:00Z").asDate().dateAdd(DT.day, 7).dateDiff(datetime("2023-08-02T00:00:00Z"))
+==>604800
+----
```

Comment:
please include the JIRA and further links to reference documentation as is form for these entries..

Reply:
added in https://github.com/apache/tinkerpop/pull/2223/commits/cc87521b7c1586aeae4b5a52f57633d628c17286